READMEv2.md

```diff
@@ -0,0 +1,213 @@
+![Versatile Data Kit](./support/images/versatile-data-kit.svg#gh-light-mode-only)
+![Versatile Data Kit](./support/images/versatile-data-kit.svg#gh-dark-mode-only)
+
+<p align="center">
+    <a href="https://github.com/vmware/versatile-data-kit/pulse" alt="Activity">
+        <img src="https://img.shields.io/github/commit-activity/m/vmware/versatile-data-kit" /></a>
+    <a href="https://github.com/vmware/versatile-data-kit/contributors" alt="Last Activity">
+        <img src="https://img.shields.io/github/last-commit/vmware/versatile-data-kit" alt="Last Activity"></a>
+    <a href="https://github.com/vmware/versatile-data-kit/blob/main/LICENSE" alt="License">
+        <img src="https://img.shields.io/github/license/vmware/versatile-data-kit" alt="license"></a>
+    <a href="https://github.com/pre-commit/pre-commit">
+        <img src="https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white" alt="pre-commit"></a>
+    <a href="https://github.com/vmware/versatile-data-kit">
+        <img src="https://gitlab.com/vmware-analytics/versatile-data-kit/badges/main/pipeline.svg" alt="build status"></a>
+    <a href="https://twitter.com/intent/tweet?text=Wow: @VDKProject">
+        <img src="https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2FVDKProject" alt="twitter"/></a>
+     <a href="https://www.youtube.com/channel/UCasf2Q7X8nF7S4VEmcTHJ0Q">
+        <img alt="YouTube Channel Subscribers" src="https://img.shields.io/youtube/channel/subscribers/UCasf2Q7X8nF7S4VEmcTHJ0Q?style=social"></a>
+
+<!-- TODO: code coverage -->
+</p>
+
+
+<p align="center">
+  <span> 🧑‍💻 Develop </span> <span> ▶️ Run </span> <span> 📊 Manage </span> <br>
+   <span>data workloads<span>
+</p>
+
+---
+
+<div align="left">
+    <span>🎯 Write shorter, more readable code. </span><br>
+</div>
+<div align="left">
+    <span>🔄 Ready-to-use data ETL/ELT patterns. </span><br>
+</div>
+<div align="left">
+    <span>🧩 Lego-like extensibility. </span><br>
+</div>
+<br>
+<div align="left">
+    <span>🚀 Single click deployment. </span><br>
+</div>
+<div align="left">
+    <span>🛠 Operate and monitor. ️</span><br>
+</div>
+
+---
+
+<h3 align="center">Introduction to the VDK SDK</h1>
```

Comment:
Something else that comes to mind. Imagine I'm trying to watch the video on data ingestion, and there are two different videos, one above, one below, also playing. It's a bit distracting.

https://github.com/vmware/versatile-data-kit/assets/91800778/02b21a11-ed84-41e1-8751-a02df43362b4

Is there a way to embed a player in the markdown, so the videos don't start all at the same time?

Reply:
The only thing I can find for now is users can disable auto play in their settings - https://github.blog/changelog/2022-05-19-option-to-prevent-animated-images-from-playing-automatically/